sql/src/test/java/io/crate/analyze/SetAnalyzerTest.java

```diff
@@ -231,37 +231,8 @@ public void testResetLoggingSetting() {
     }
 
     @Test
-    public void testSetLicenseAcceptValidSymbolValues() throws Exception {
-        SetLicenseAnalyzedStatement analysis = analyze("SET LICENSE expirationDate='2020-12-31T01:02:03.123', issuedTo='me', signature='XXX'");
-        assertThat(analysis.expirationDateSymbol(),
-            is(io.crate.expression.symbol.Literal.of("2020-12-31T01:02:03.123")));
-        assertThat(analysis.issuedToSymbol(),
-            is(io.crate.expression.symbol.Literal.of("me")));
-        assertThat(analysis.signatureSymbol(),
-            is(io.crate.expression.symbol.Literal.of("XXX")));
-    }
-
-    @Test
-    public void testSetLicenseWrongNumberOfSettingsThrowsIllegalArgumentException() throws Exception {
-        expectedException.expect(IllegalArgumentException.class);
-        expectedException.expectMessage("Invalid number of settings for SET LICENSE. " +
-                                        "Please provide the following settings: [issuedto, signature, expirationdate]");
-        analyze("SET LICENSE issuedTo='me', signature='XXX'");
-    }
-
-    @Test
-    public void testSetLicenseInvalidSettingNameThrowsIllegalArgumentException() throws Exception {
-        expectedException.expect(IllegalArgumentException.class);
-        expectedException.expectMessage("Invalid setting 'expirationdat' for SET LICENSE. " +
-                                        "Please provide the following settings: [issuedto, signature, expirationdate]");
-        analyze("SET LICENSE expirationDat='2020-12-31T01:02:03.123', issuedTo='me', signature='XXX'");
-    }
-
-    @Test
-    public void testSetLicenseMissingSettingThrowsNullPointerException() throws Exception {
-        expectedException.expect(NullPointerException.class);
-        expectedException.expectMessage("Missing setting for SET LICENSE. " +
-                                        "Please provide the following settings: [issuedto, signature, expirationdate]");
-        analyze("SET LICENSE issuedTo='you', issuedTo='me', signature='XXX'");
+    public void testSetLicenseValidCase() throws Exception {
```

Comment:
shall we maybe rename this to testSetLicense - not sure "validCase" adds too much value here